### PR TITLE
Adding release notes declaring changes related to CORS

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
Adding Changes related to CORS to release notes, to notify users about ability to test the new configuration before Che 6.16 release

Merge only after succesful merge of related PR https://github.com/eclipse/che/pull/12046

### CODE REVIEW

File: release-notes/release-6.15.0.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+1) use `CHE_CORS_ENABLED=false` environment variable for Che deployment to disable CORS on WS Master.
+2) use `CHE_WSAGENT_CORS_ALLOWED__ORIGINS=<wsmaster-domain>` to set the default allowed origin of WS Agent CORS pointing to Domain of WS Master (replace `<wsmaster-domain>` with actual Domain value)
+Setting origin for WS Agent would be later done automatically in PR for Upgrading Tomcat.
+
+**If you discover an regression related to this configuration (for example, certain cross origin requests are not working in your Che installation).
+Please reach out to us describing the issues you have, so we may address them before Che 6.16 release.**
+
+Additionally, here is the full list of environment vairables, that are introduced in 6.15 for configuring CORS filter (https://github.com/eclipse/che/issues/12058), should you want to try other configurations:
+You can use them, to tryo your own CORS configuration for WS Master (by applying following variables to Che deployemt), or WS Agent (by applying them to Che Workspace configuration)
```
Comment (sleshchenko) [inline release-notes/release-6.15.0.md]: typos in `tryo`, `deployemt`

**Review by sleshchenko — APPROVED** [review]
LGTM

File: release-notes/release-6.15.0.md (hunk 1 above)
Comment (mkuznyetsov) [reply release-notes/release-6.15.0.md]: thank you

**Review by skabashnyuk — APPROVED** [review]

**Comment by mkuznyetsov** [comment]
@dneary can we make sure this information ends up in release notes?

**Comment by dneary** [comment]
> @dneary can we make sure this information ends up in release notes?

Of course! I may make some edits to content to present the information for an audience unfamiliar with Che internals and/or CORS.

**Pull request merged**